web/packages/teleterm/src/ui/Search/pickers/ResultList.tsx

```diff
@@ -157,6 +154,11 @@ const InteractiveItem = styled(NonInteractiveItem)`
     cursor: pointer;
     background: ${props => props.theme.colors.levels.elevated};
   }
+
+  background: ${props =>
+    props.$active
+      ? props.theme.colors.levels.elevated
+      : props.theme.colors.levels.surface};
```

Comment:
I fixed it in my PR too, but without duplicating the inactive color! :D  https://github.com/gravitational/teleport/pull/24447/commits/6cac00622de0e6379f2ed6fe1baf7557325f2975#diff-625475f04162a762fc09ebe91356eccd77ab58991ccced828ea67b821f176928R159

Reply:
I'll leave it here and then we'll solve the conflict down the road. ;)